typings/mapping.d.ts

```diff
@@ -0,0 +1,14 @@
+/** Generated from {@link ../tools/create-mapping.sh} */
+import { LitElement } from 'lit';
```

Comment:
This import is needed to extend the HTMLElementTagNameMap. Not sure if there is a better way to do this.